### PR TITLE
Single, group, target fields changes in volunteers

### PR DESCRIPTION
**Pull request by chrysalag**
# [Story in Trello](https://trello.com/c/qBqPjSTA/58-uat-feedback-f%C3%BCr-freiwilligeneins%C3%A4tze-einzelbegleitung-gruppenangebot-zielgruppen)

### Done for approval?

* [x] Acceptance criteria are met
* [x] Code quality checks are green
* ~[ ] Readme updated if needed~
* [x] Story under test
* ~[ ] Mobile works~
* [x] Seeds created
* [x] Translated
* [x] Code is reviewed

### Done after the merge?

* [ ] Deployed to staging after the merge
* [ ] Tested manually on staging

### CODE REVIEW

**Comment by chrysalag** [comment]
Travis is red because of a test that randomly fails.

File: app/controllers/concerns/volunteer_attributes.rb [hunk 1]
```diff
@@ -5,12 +5,12 @@ module VolunteerAttributes
...
-        :family, :kid, :sport, :creative, :music, :culture, :training,
-        :german_course, :adults, :teenagers, :children, :region, :state,
-        :working_percent, :volunteer_experience_desc, :region_specific,
-        contact_attributes, language_skills_attributes, schedules_attributes
+        :profession, :education, :motivation, :experience, :expectations, :strengths,
+        :skills, :interests, :state, :man, :woman, :family, :kid, :sport, :creative,
+        :music, :culture, :training, :german_course, :teenagers, :children, :state,
+        :working_percent, :volunteer_experience_desc, :dancing, :health, :cooking,
+        :excursions, :women, :unaccompanied, :zurich, :other_offer, :other_offer_desc,
+        :own_kids, contact_attributes, language_skills_attributes, schedules_attributes
```
Comment (nuritnt) [inline app/controllers/concerns/volunteer_attributes.rb]: family instead of own kids? like the label?

File: app/models/volunteer.rb [hunk 2]
```diff
@@ -73,12 +73,11 @@ def self.state_collection
...
-  REGIONS = [:city, :region, :canton].freeze
-  SINGLE_ACCOMPANIMENTS = [:man, :woman, :family, :kid].freeze
-  GROUP_ACCOMPANIMENTS = [:sport, :creative, :music, :culture, :training, :german_course].freeze
+  SINGLE_ACCOMPANIMENTS = [:man, :woman, :family, :kid, :unaccompanied].freeze
+  GROUP_ACCOMPANIMENTS = [:sport, :creative, :music, :culture, :training, :german_course,
+                          :dancing, :health, :cooking, :excursions, :women, :teenagers,
+                          :children, :other_offer].freeze
   REJECTIONS = [:us, :her, :other].freeze
-  TARGET_GROUPS = [:adults, :teenagers, :children].freeze
   GENDERS = [:female, :male].freeze
```
Comment (nuritnt) [inline app/models/volunteer.rb]: just stumbling upon this, regarding my story from last week. I thought I changed all `genders` to `salutation(mrs/mr)`. is this constant used in matching? or even not used anymore?

File: app/views/volunteers/_form.html.slim [hunk 3]
```diff
@@ -56,6 +56,8 @@
   .col-xs-4
     = f.input :working_percent
   .col-xs-12
+    = f.input :own_kids
```
Comment (nuritnt) [inline app/views/volunteers/_form.html.slim]: renaming to family?

File: app/views/volunteers/_form.html.slim [hunk 4]
```diff
@@ -83,17 +88,12 @@ h3= t_attr(:single_accompaniment, Volunteer)
...
+    = f.input :other_offer_desc
 
-h3= t('your_region')
 .row
-  .col-xs-4.col-md-2
-    = f.input :region, collection: Volunteer::REGIONS, as: :radio_buttons, label: false
-  .col-xs-8.col-md-10
-    = conditional_field f, :region_specific, :region, :region
+  .col-xs-12
+    = f.input :zurich
```
Comment (nuritnt) [inline app/views/volunteers/_form.html.slim]: I would move this even above the description field - directly after line 88

File: app/views/volunteers/show.html.slim [hunk 5]
```diff
@@ -58,6 +58,9 @@ h3= t('personal_background')
         td= t_attr(:profession)
         td= @volunteer.profession
       tr
+        td= t_attr(:own_kids)
```
Comment (nuritnt) [inline app/views/volunteers/show.html.slim]: renaming to family?

File: app/controllers/concerns/volunteer_attributes.rb (hunk 1 above)
Comment (chrysalag) [inline app/controllers/concerns/volunteer_attributes.rb]: We already have a voluntary attribute named family.

File: app/models/volunteer.rb (hunk 2 above)
Comment (chrysalag) [inline app/models/volunteer.rb]: It's not used anymore. The Salutation collection is in a concern. This should be removed, yes.

**Pull request merged**